src/selectors/per-thread/thread.js

```diff
@@ -132,6 +132,47 @@ export function getThreadSelectorsPerThread(threadIndex: ThreadIndex): * {
     }
   );
 
+  /**
+   * This selector returns the offset to add to a sampleIndex when accessing the
+   * base thread, if your thread is a range filtered thread (all but the base
+   * `getThread` or the last `getPreviewFilteredThread`).
+   */
+  const getSampleIndexOffsetFromCommittedRange: Selector<number> = createSelector(
+    getThread,
+    ProfileSelectors.getCommittedRange,
+    ({ samples }, { start, end }) => {
+      const [beginSampleIndex] = ProfileData.getSampleIndexRangeForSelection(
+        samples,
+        start,
+        end
+      );
+      return beginSampleIndex;
+    }
+  );
+
+  /**
+   * This selector returns the offset to add to a sampleIndex when accessing the
+   * base thread, if your thread is the preview filtered thread.
+   */
+  const getSampleIndexOffsetFromPreviewRange: Selector<number> = createSelector(
```

Comment:
I'm confused, this does not appear to be used anywhere.

Reply:
It's used in https://github.com/firefox-devtools/profiler/pull/2154/files#diff-252a5225ae887d8bfc3c24bbf7d1a48fR151

Reply:
Ah, not sure how I missed that.